### ISSUE TITLE
Error finding implementation of: ...ReminderController

### ISSUE THREAD

**Issue by rogersm** [orbit/orbit#311]
Doing some simple tests with Orbit-core 1.10.1, the system works, but I got a some exceptions, like this one:

```java

[OrbitThread-25] ERROR cloud.orbit.actors.runtime.ClassPathSearch - Error finding implementation of: cloud.orbit.actors.runtime.ReminderController as cloud.orbit.io.StringBuilderWriter
java.lang.UnsupportedOperationException
	at cloud.orbit.core.shaded.org.objectweb.asm.ClassVisitor.visitNestMemberExperimental(ClassVisitor.java:248)
	at cloud.orbit.core.shaded.org.objectweb.asm.ClassReader.accept(ClassReader.java:651)
	at cloud.orbit.core.shaded.org.objectweb.asm.ClassReader.accept(ClassReader.java:391)
	at cloud.orbit.actors.runtime.ClassPathSearch.getClassInfo(ClassPathSearch.java:157)
	at cloud.orbit.actors.runtime.ClassPathSearch.getClassInfo(ClassPathSearch.java:174)
	at cloud.orbit.actors.runtime.ClassPathSearch.lambda$doFind$4(ClassPathSearch.java:258)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1631)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
	at cloud.orbit.actors.runtime.ClassPathSearch.doFind(ClassPathSearch.java:308)
	at cloud.orbit.actors.runtime.ClassPathSearch.findImplementation(ClassPathSearch.java:217)
	at cloud.orbit.actors.runtime.ClassPathSearch.findImplementation(ClassPathSearch.java:206)
	at cloud.orbit.actors.runtime.LazyActorClassFinder.findActorImplementation(LazyActorClassFinder.java:44)
	at cloud.orbit.actors.Stage.canActivateActor(Stage.java:1559)
	at cloud.orbit.actors.runtime.Hosting.canActivate(Hosting.java:184)
	at cloud.orbit.actors.runtime.Hosting$ObjectInvoker.invoke(Hosting$ObjectInvoker.java)
	at cloud.orbit.actors.runtime.ObjectInvoker.safeInvoke(ObjectInvoker.java:62)
	at cloud.orbit.actors.runtime.DefaultInvocationHandler.doInvoke(DefaultInvocationHandler.java:71)
	at cloud.orbit.actors.runtime.SimpleInvocationHandler.invoke(SimpleInvocationHandler.java:99)
	at cloud.orbit.actors.runtime.Execution.performInvocation(Execution.java:163)
	at cloud.orbit.actors.runtime.Execution.lambda$onInvocation$0(Execution.java:86)
	at cloud.orbit.actors.runtime.ObserverEntry.lambda$run$0(ObserverEntry.java:64)
	at cloud.orbit.actors.runtime.InternalUtils.safeInvoke(InternalUtils.java:192)
	at cloud.orbit.actors.concurrent.WaitFreeMultiExecutionSerializer.lambda$offerJob$2(WaitFreeMultiExecutionSerializer.java:90)
	at cloud.orbit.actors.runtime.InternalUtils.safeInvoke(InternalUtils.java:192)
	at cloud.orbit.actors.concurrent.WaitFreeExecutionSerializer.lambda$executeSerialized$0(WaitFreeExecutionSerializer.java:88)
	at cloud.orbit.actors.concurrent.WaitFreeExecutionSerializer.tryExecute(WaitFreeExecutionSerializer.java:134)
	at cloud.orbit.actors.concurrent.WaitFreeExecutionSerializer.lambda$tryExecute$1(WaitFreeExecutionSerializer.java:154)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)

```

Is this something expected?

**Comment by JoeHegarty** [comment]
Looks like this is a bug with Orbit on JDK11.
Can you try setting `basePackages` when you create the Stage? That should work around it for now.

We will fix the error ASAP.

**Comment by JoeHegarty** [comment]
Resolved in v1.10.2

https://github.com/orbit/orbit/releases/tag/v1.10.2
